presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -361,6 +365,10 @@ callArgument
     | identifier '=>' expression    #namedArgument
     ;
 
+privilege
+    : SELECT | DELETE | INSERT | value=identifier
```

Comment:
Done in https://github.com/Teradata/presto/commit/f0ccf281bc9f13601cc1826efe262a57016e1675